### PR TITLE
windows unification

### PR DESCRIPTION
**Pull request by margirova**
-not tested yet
you may notice, that in case of session I sometimes add 'window', sometimes not. This inconsistency happens because I have the problem with the term "window" itself. In case of timewindow we have row_dimension per window(time_dimension), but in case of sessions we have session(row)_dimension per window(whatever_dimension_we_specify_but_not_really_anything_specific_and_continuos). So I am open for suggestions..

### CODE REVIEW

**Comment by margirova** [comment]
- added storage support
- renamed window to capture. Does it sound cool?

**Comment by margirova** [comment]
- removed multiple matrixes
- updated everything
- tested
theoretically it's ready to merge

File: bspump/analyzer/window.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
+import time
```
Comment (ateska) [inline bspump/analyzer/window.py]: This is not used anywhere ... is this still surviving?

File: bspump/analyzer/aggregation.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+###
+
+L = logging.getLogger(__name__)
+
+###
+
+
+
+class Aggregation(object):
```
Comment (ateska) [inline bspump/analyzer/aggregation.py]: Not sure if the name is right ... aggregation has a different meaning ... people will do other than aggregations with this.
I would go for `MatrixContainer` etc.

**Pull request merged**